src/server.c

```diff
@@ -150,62 +116,16 @@ void
 sig_handler(int sig) {
     if (force_exit)
         exit(EXIT_FAILURE);
-
-    char sig_name[20];
-    get_sig_name(sig, sig_name);
-    lwsl_notice("received signal: %s (%d), exiting...\n", sig_name, sig);
+    lwsl_notice("received signal: %s (%d), exiting...\n", strsignal(sig), sig);
```

Comment:
You may get `received signal: SIGHUP (1)` or `received signal: SIGHUP (1) (1)` on different os, but the later one may confuse user.